tui/tui.go

```diff
@@ -103,6 +103,12 @@ func (tui *Tui) Run(md *model.SystemInstall, rootDir string, options args.Args)
 		tui.model.CopySwupd = true
 	}
 
+	// Check if we can boot EFI
+	if !utils.HostHasEFI() {
```

Comment:
Why do we do this check only for tui?

Reply:
We check in both the TUI and the GUI https://github.com/clearlinux/clr-installer/blob/master/gui/window.go#L195
The problem being fixed here is that in the TUI the EFI check was too late in the code; it is now moved before the partition information is processed.